README.md

```diff
@@ -232,6 +232,12 @@ Global hook which will run immediately before send API response.
 
 Global hook which will run immediately after sent API response back to client.
 
+##### log\_error\_on\_finish (global)
+
+    log_error_on_finish => [sub { my ($d, $code, $error_msg) = @_; ... }, sub {...}]
```

Comment:
ditto, also it seems it accepts only a single coderef rather than a series of coderefs.

Reply:
same here